.version

```diff
@@ -0,0 +1 @@
+2.4.0
```

Comment:
I assume we're not going to try to keep this up to date with the product stream version, right? We'd have to predict what the version of the product stream would be when it's generated to include this version of the docs.

Should we just cut off the patch version and make it 2.4? If we did that, would we have collisions of version numbers because the `-1` is always the same on stable builds?

Reply:
That's correct, was not planning on keeping this in sync with the product stream.

I'd be OK with dropping the third digit if you think it would cut down on confusion (specifically in this area, wondering if you need to match the docs version to the product stream version). While it could cause collisions, so would rebuilding the RPM or checking in a change and forgetting to bump the version.

It also would mean we no longer need to have a `docs-sat-latest.noarch.rpm`, since for 2.4 you would always just download `docs-sat-2.4.noarch.rpm`. It's possible that's actually a good thing, since we could reduce some of the complexity with how we are publishing the RPM.

Reply:
I think optimally we'd have the x.y version of the SAT product stream in this repo in addition to some versioning on the docs themselves.  Perhaps just a commit counter kind of thing would be sufficient. The way we had our docs versioned before with the x.y version of the SAT product stream with commit and timestamp information appended worked okay to differentiate the different versions of the docs.

Reply:
I think of those two options (commit counter vs date/timestamp) I'd probably prefer doing a commit counter type thing.

Reply:
Where would we start the commit counter from? Would it reset on each `release/` branch?

Reply:
I think we would have to create a `v2.4` tag here on main (e.g. at the current HEAD of main), and then count commits after it. When we create the `release/2.4` branch from main, that tag should remain in the history and continue to work, then we would have to create a `v2.5` tag on main so that subsequent builds of main were numbered 2.5.x.

Reply:
Actually that doesn't totally work when our dev branches are based off integration rather than main.

Reply:
Let's discuss and see if we can reach a decision at scrum on Monday.

Reply:
We discussed in scrum, and this is our conclusion.

* We will tag the commit that switches the docs to target a new version of the SAT product stream. This tag will be the major and minor version of the SAT product stream to which the docs apply. E.g. 2.4
* We will consider whether we can remove the `.version` file and just use a `git describe` command to find the most recent tag.
* We will pass a release version to the `getRpmRevision` shared library function as the value of the `buildRef` argument. This release version will be the count of commits since the last SAT version tag. This will be used as the "Release" of the RPM. This will be returned as-is on stable builds and will have commit hash and timestamp on unstable builds. E.g., a stable build will be of the form docs-sat-2.4-12, and unstable will be docs-sat-2.4-12~dev-branch-<timestamp>-<commit-hash>

Reply:
I've addressed this in ba9eae065a15a3e95dcfc94ac77f91fd875e7a27, and also squashed the commits.